src/Servers/IIS/AspNetCoreModuleV2/InProcessRequestHandler/dllmain.cpp

```diff
@@ -129,7 +134,20 @@ CreateApplication(
             std::unique_ptr<InProcessOptions> options;
             THROW_IF_FAILED(InProcessOptions::Create(*pServer, pSite, *pHttpApplication, options));
             // Set the currently running application to a fake application that returns startup exceptions.
-            auto pErrorApplication = std::make_unique<StartupExceptionApplication>(*pServer, *pHttpApplication, g_hServerModule, options->QueryDisableStartUpErrorPage(), hr, std::move(g_errorPageContent));
+            auto pErrorApplication = std::make_unique<StartupExceptionApplication>(*pServer,
+                *pHttpApplication,
+                options->QueryDisableStartUpErrorPage(),
+                hr,
+                FILE_UTILITY::GetHtml(g_hServerModule,
+                    IN_PROCESS_RH_STATIC_HTML,
+                    500i16,
+                    30i16,
+                    "ANCM In-Process Start Failure",
+                    "<ul><li>The application failed to start</li><li>The application started but then stopped</li><li>The application started but threw an exception during startup</li></ul>",
+                    std::move(g_errorPageContent)),
```

Comment:
Don't `std::move` a global. And since you should change the `GetHtml` function based on my other comment this wont be needed to save the copy.